### DISCUSSION TITLE
The CNAME is already taken

### DISCUSSION THREAD

**Discussion by sergiks** [community/community#22188] [Other Feature Feedback, Questions, & Ideas]
<p>I used to have a repo for a static web site years ago. Directed custom domain to it, all worked fine.</p>
<p>After a while I deleted that repo.</p>
<p>Few years later (now) I created a new repo, under an Organization, to publish a static web site for that domain again.</p>
<p>Error appears: “The CNAME … is already taken”</p>
<p>Tried removing the CNAME file from my new repo. The site got published under its <code>.github.io</code> subdomain.</p>
<p>Restored the CNAME file — the error is back, and the naked Custom domain displays 404 again.</p>
<p>Please advice, who can I contact to ask to remove the “stuck” cname record from GitHub internals. Or maybe I should create a new repo with the same name as the initial one that was deleted?</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/sergiks">@sergiks</a>, welcome to the GitHub Support Community! This error usually happens when your custom domain has been attached to someone else’s Pages site in error, or the previous owner of this domain used it to host a Pages site before it was bought by you.</p>
<p>You’ll need to reach out to the GitHub Support team for direct private help with this as it will require adding a verification code as a DNS record to your domain to prove your ownership. Make sure to include the domain you’re attempting to attach to your Pages site in the message so they can help you quickly. You should be able to send a message to the support team at the following link and choose the “Help with a Pages domain or HTTPS certificate” option:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://support-assets.github.com/assets/favicon-3dcd2c977a91b2e3f396b300c5ff0d572295ca632ddb7ba1adcacc1ea68dd5c0.ico" class="site-icon" width="32" height="32">
      <a href="https://support.github.com/request" target="_blank" rel="noopener">support.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://support.github.com/request" target="_blank" rel="noopener">GitHub Support</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by sergiks** [comment]
<p>Thanks!</p>
<p>Indeed, this helped. I opened a support ticket, they promptly responded. I had to verify domain ownership by adding a TXT DNS record provided, and in a matter of 1 hour the CNAME was released and my new Pages repo was working with my custom domain.</p>
<p>Case closed )</p>
<p>Thanks for support!</p>
